README.md

```diff
@@ -33,7 +33,6 @@
     [`OptRef`](http://avsystem.github.io/scala-commons/api/com/avsystem/commons/misc/OptRef.html) (implemented as value
     classes)
   * [Components](docs/Components.md) and Dependency Injection library
-* `commons-redis` - [Scala driver for Redis](docs/RedisDriver.md)
```

Comment:
suggestion: If we're keeping the Redis docs (I noticed you added deprecation notice in `RedisDriver.md`) until we remove the driver code entirely, IMO we should keep the entrypoint to its docs too.